### PR TITLE
JGC-238 fix folder move/copy with parentheses

### PR DESCRIPTION
**Pull request by Postyy**

### CODE REVIEW

File: artifactory_test.go [hunk 1]
```diff
@@ -326,6 +327,60 @@ func TestArtifactoryDirectoryCopyUsingWildcard(t *testing.T) {
 	cleanArtifactoryTest()
 }
 
+func TestArtifactoryCopyFilesNameWithBracket(t *testing.T) {
```
Comment (eyalbe4) [inline artifactory_test.go]: Shouldn't the function name be **TestArtifactoryCopyFilesNameWithParenthesis**?
Same goea for the *TestArtifactoryUploadFilesNameWithBracket* and *TestArtifactoryDownloadFilesNameWithBracket* functions.

File: artifactory_test.go [hunk 2]
```diff
@@ -326,6 +327,60 @@ func TestArtifactoryDirectoryCopyUsingWildcard(t *testing.T) {
...
+	artifactoryCli.Exec("upload", "--spec="+specFile)
+	searchPath, err := tests.CreateSpec(tests.SearchAllRepo1)
+	if err != nil {
+		t.Error(err)
+	}
+	isExistInArtifactory(tests.GetUploadFileNameWithParentheses(), searchPath, t)
+
+	cleanArtifactoryTest()
+}
+func TestArtifactoryDownloadFilesNameWithBracket(t *testing.T) {
```
Comment (eyalbe4) [inline artifactory_test.go]: Missing new-line before this function.

File: artifactory_test.go [hunk 3]
```diff
@@ -326,6 +327,60 @@ func TestArtifactoryDirectoryCopyUsingWildcard(t *testing.T) {
...
+
+	searchPath, err := tests.CreateSpec(tests.SearchRepo2)
+	if err != nil {
+		t.Error(err)
+	}
+	isExistInArtifactory(tests.GetCopyFileNameWithParentheses(), searchPath, t)
+
+	cleanArtifactoryTest()
+}
+func TestArtifactoryUploadFilesNameWithBracket(t *testing.T) {
```
Comment (eyalbe4) [inline artifactory_test.go]: Missing new-line before this function.

File: utils/tests/artifactoryconsts.go [hunk 4]
```diff
@@ -56,6 +57,7 @@ const (
 	GoLocalRepositoryConfig                = "go_local_repository_config.json"
 	RepoDetailsUrl                         = "api/repositories/"
 	CopyItemsSpec                          = "copy_items_spec.json"
+	CopyFileWithParenthesesSpec            = "upload_file_with_parentheses.json"
```
Comment (eyalbe4) [inline utils/tests/artifactoryconsts.go]: Should this constant name be *UploadFileWithParenthesesSpec*?

File: utils/tests/artifactoryconsts.go [hunk 5]
```diff
@@ -382,6 +418,26 @@ func GetExtractedDownload() []string {
 		filepath.Join(Out, "concurrent.tar.gz"),
 	}
 }
+func GetFileWithParenthesesDownload() []string {
```
[the remark this answers is not in the document]
Comment (Postyy) [reply utils/tests/artifactoryconsts.go]: Done.

File: artifactory_test.go (hunk 1 above)
Comment (Postyy) [reply artifactory_test.go]: Done.

File: artifactory_test.go (hunk 2 above)
Comment (Postyy) [reply artifactory_test.go]: Done.

File: artifactory_test.go (hunk 3 above)
Comment (Postyy) [reply artifactory_test.go]: Done.

File: utils/tests/artifactoryconsts.go (hunk 4 above)
Comment (Postyy) [reply utils/tests/artifactoryconsts.go]: Yes, Done.

**Pull request merged**